javascript/react/src/components/ManageWorkflow.types.ts

```diff
@@ -2,11 +2,15 @@
  * Copyright (c) Microsoft. All rights reserved.
  * Licensed under the MIT license. See LICENSE file in the project.
  */
-import type { TableContainer } from '@datashaper/tables'
 import type { Workflow } from '@datashaper/workflow'
 
 import type { TransformModalProps } from '../index.js'
 
+export enum WorkflowOrder {
+	FirstOnTop = 'first on top',
```

Comment:
Suggest not using spaces for enum values in case they're used as keys where spaces cause a problem. You can mash together `firstontop` or use dashes `first-on-top`